[2024-12-24T17:19:43.322+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-24T17:19:43.357+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_connect.task_get_conn scheduled__2024-12-23T08:00:00+00:00 [queued]>
[2024-12-24T17:19:43.369+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_connect.task_get_conn scheduled__2024-12-23T08:00:00+00:00 [queued]>
[2024-12-24T17:19:43.369+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-12-24T17:19:43.395+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): task_get_conn> on 2024-12-23 08:00:00+00:00
[2024-12-24T17:19:43.402+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=71) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-24T17:19:43.403+0000] {standard_task_runner.py:63} INFO - Started process 72 to run task
[2024-12-24T17:19:43.404+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_connect', 'task_get_conn', 'scheduled__2024-12-23T08:00:00+00:00', '--job-id', '1067', '--raw', '--subdir', 'DAGS_FOLDER/dag_connect.py', '--cfg-path', '/tmp/tmp73ol7zfo']
[2024-12-24T17:19:43.405+0000] {standard_task_runner.py:91} INFO - Job 1067: Subtask task_get_conn
[2024-12-24T17:19:43.451+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_connect.task_get_conn scheduled__2024-12-23T08:00:00+00:00 [running]> on host 87a1e11545b0
[2024-12-24T17:19:43.564+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_connect' AIRFLOW_CTX_TASK_ID='task_get_conn' AIRFLOW_CTX_EXECUTION_DATE='2024-12-23T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-23T08:00:00+00:00'
[2024-12-24T17:19:43.565+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-24T17:19:43.592+0000] {base.py:84} INFO - Using connection ID 'my_postgres' for task execution.
[2024-12-24T17:19:43.592+0000] {logging_mixin.py:188} INFO - Host: jdbc:postgresql://localhost:5436/de_course_db
[2024-12-24T17:19:43.593+0000] {logging_mixin.py:188} INFO - Login: postgres
[2024-12-24T17:19:43.593+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-12-24T17:19:43.594+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-24T17:19:43.606+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_connect, task_id=task_get_conn, run_id=scheduled__2024-12-23T08:00:00+00:00, execution_date=20241223T080000, start_date=20241224T171943, end_date=20241224T171943
[2024-12-24T17:19:43.662+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-12-24T17:19:43.677+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-24T17:19:43.678+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
